src/application/clients/crux-wallet-client.ts

```diff
@@ -133,15 +136,34 @@ export class CruxWalletClient {
         if (!cruxUser) {
             throw ErrorHelper.getPackageError(null, PackageErrorCode.UserDoesNotExist);
         }
-        const assetId = this.cruxAssetTranslator.symbolToAssetId(walletCurrencySymbol);
-        if (!assetId) {
-            throw ErrorHelper.getPackageError(null, PackageErrorCode.AssetIDNotAvailable);
+        const addressResolver = new CruxAddressResolver({
```

Comment:
This function should not change

Reply:
as discussed, not querying the addressMap with asset